### DISCUSSION TITLE
Variable sharing between SQL cells and C#

### DISCUSSION THREAD

**Discussion by LSTANCZYK** [dotnet/interactive#1733] [Q&A] [answered]
I am looking at `Variable sharing.ipynb`
Is it possible to take SQL cell result set (from SQL subkernel) and pass it to C# as a variable (say DataFrame or dynamic/Expando object)?

**Comment by jonsequitur** [comment, marked as answer]
Yes it is. It's currently shared as a `TabularDataResource` but we're looking into changing this to a `DataFrame` or `IDataView`.

![image](https://user-images.githubusercontent.com/547415/141514940-24893688-e26a-46d4-96f4-ea2014c515b8.png)
